i18n/en.json

```diff
@@ -1,27 +1,27 @@
 {
   "about.buildnumber": "Build Number:",
   "about.cloudEdition": "Cloud",
-  "about.copyright": "Copyright 2015 - {currentYear} Mattermost, Inc. All rights reserved",
+  "about.copyright": "Copyright 2018 - {currentYear} Riff Analytics, Inc. All rights reserved",
   "about.database": "Database:",
   "about.date": "Build Date:",
   "about.dbversion": "Database Schema Version:",
-  "about.enterpriseEditione1": "Enterprise Edition",
-  "about.enterpriseEditionLearn": "Learn more about Enterprise Edition at ",
-  "about.enterpriseEditionSst": "High trust messaging for the enterprise",
-  "about.enterpriseEditionSt": "Modern communication from behind your firewall.",
+  "about.enterpriseEditione1": "",
+  "about.enterpriseEditionLearn": "",
+  "about.enterpriseEditionSst": "",
+  "about.enterpriseEditionSt": "",
   "about.hash": "Build Hash:",
   "about.hashee": "EE Build Hash:",
   "about.hashwebapp": "Webapp Build Hash:",
   "about.licensed": "Licensed to:",
-  "about.notice": "Mattermost is made possible by the open source software used in our [server](!https://about.mattermost.com/platform-notice-txt/), [desktop](!https://about.mattermost.com/desktop-notice-txt/) and [mobile](!https://about.mattermost.com/mobile-notice-txt/) apps.",
+  "about.notice": "Riff Edu is made possible by the open source software used in our [server](!https://about.mattermost.com/platform-notice-txt/).",
   "about.privacy": "Privacy Policy",
-  "about.teamEditionLearn": "Join the Mattermost community at ",
-  "about.teamEditionSt": "All your team communication in one place, instantly searchable and accessible anywhere.",
-  "about.teamEditiont0": "Team Edition",
-  "about.teamEditiont1": "Enterprise Edition",
+  "about.teamEditionLearn": "Learn more at ",
+  "about.teamEditionSt": "A centralized collaboration hub enabling instructor and student success.",
+  "about.teamEditiont0": " ",
```

Comment:
everywhere else you changed the text to an empty string, here you change it to a single space?

Reply:
Yeaaaaah empty space was to suppress the message, empty string defaults to the default message. A little awkward but I didn't want to change the about_build_modal.tsx file in places I didn't have to. The rest of the empty strings are ones I forgot to change back before I realized empty string would default to the `FormattedMessage`'s default message, but ultimately doesn't really matter since they're for versions we never display anyway